EditView.vala

```diff
@@ -33,6 +33,10 @@ class EditView: Gtk.DrawingArea, Gtk.Scrollable {
 	public string label { private set; get; }
 	public bool has_unsaved_changes { private set; get; }
 
+	static construct {
+		set_css_name("xieditview");
```

Comment:
This requires GTK+ 3.20 so we'd have to bump the required GTK+ version which is currently GTK+ 3.18 which would mean Ubuntu 16.04 is no longer supported.